src/Corax/IndexWriter.cs

```diff
@@ -305,6 +306,10 @@ void Insert(ReadOnlySpan<byte> value)
 
             void AnalyzeInsert(Span<byte> wordsBuffer, Span<Token> tokens, ReadOnlySpan<byte> value)
             {
+                if (wordsBuffer.Length < value.Length)
+                {
+                    throw new Exception($"Passed value with length: {value.Length}. Contains {Encodings.Utf8.GetString(value)} but buffer is only {wordsBuffer.Length}");
```

Comment:
Let's not throw raw `Exception`

Reply:
Deleted when implemented growable buffer.